pom.xml

```diff
@@ -44,7 +44,7 @@
         <dependency>
             <groupId>software.amazon.payloadoffloading</groupId>
             <artifactId>payloadoffloading-common</artifactId>
-            <version>1.0.0</version>
+            <version>1.1.1</version>
```

Comment:
Latest version is 2.1.1. Are you sure you want to bump to 1.1.1 ?

Reply:
Would you recommend bumping to 2.1.2?

Reply:
I checked with the Adam Bchouti, who did the latest changes in the common lib. We need to use version 1.1.1 because of a breaking change introduced bu migation to sdk 2.0